meta/main.yml

```diff
@@ -1,19 +1,23 @@
 ---
 galaxy_info:
-  author: Insert author(s) here
-  description: Insert description here
+  author: haslersn
+  description: containerd; see https://github.com/containerd/containerd
   company: stuvus (https://stuvus.uni-stuttgart.de)
 
-  license: CC-BY-SA
+  license: MIT
 
   min_ansible_version: 2.5
 
   platforms:
     - name: Debian
```

Comment:
In the README you state Ubuntu is laso supported. Here you don't. What is the case now?

Reply:
Fixed.